core/modules/loader/LoaderMsg.h

```diff
@@ -60,7 +60,7 @@ class LoaderMsg {
         KEY_INSERT_REQ,        // Insert a new key with info. MSG_RECEIVED + KEY_INFO
         KEY_INSERT_COMPLETE,   // Key has been inserted and logged.
         KEY_INFO_REQ,          // Request info for a single key.
-        KEY_INFO,              // Information about a specific key. (includes file id and row)
+        KEY_LOOK,              // Information about a specific key. (includes file id and row)
```

Comment:
Based on the comment to this value `KEY_INFO` was a more appropriate identifier. Could the comment be made consistent with the new name?